Tweetinvi.Controllers/Properties/Resources.cs

```diff
@@ -430,6 +430,11 @@ internal static class Resources
         public static string Tweet_Get = "https://api.twitter.com/1.1/statuses/show.json?id={0}&include_my_retweet=true";
 
         /// <summary>
+        ///   Looks up a localized string similar to https://api.twitter.com/1.1/statuses/retweeters/ids.json?id={0}&stringify_ids={1}.
+        /// </summary>
+        public static string Tweet_GetRetweeters = "https://api.twitter.com/1.1/statuses/retweeters/ids.json?id={0}&stringify_ids={1}";

```

Comment:
As mentioned below, the resources string should no longer have parameters added directly. Instead it should be `https://api.twitter.com/1.1/statuses/retweeters/ids.json` and the `QueryGenerator` will use `.AddQueryParameter` to add the count parameter to the query. Also `Stringify` should never be part of the query.